services/blockchain-connector/shared/utils/regex.js

```diff
@@ -16,9 +16,13 @@
 const BLS_KEY = /^\b[a-fA-F0-9]{96}\b$/;
 const CHAIN_ID = /^\b[a-fA-F0-9]{8}\b$/;
 const PROOF_OF_POSSESSION = /^\b[a-fA-F0-9]{192}\b$/;
+const CHAIN_ACCOUNT_KEY_NOT_FOUND_ERROR = /^Key [a-f0-9]{20} does not exist\.$/;
+const STARTER_KEY_NOT_FOUND_ERROR = /^Key [a-f0-9]{52} does not exist\.$/;
```

Comment:
Please use a single regex.
```suggestion
const KEY_NOT_EXIST = /^Key [a-f0-9]+ does not exist\.$/;
```